docker-compose.yml

```diff
@@ -1,9 +1,6 @@
 services:
   webserver:
-    container_name: webserver
-    build:
```

Comment:
Just a question here. Why would we not want the image to build? Makes the test faster?

Reply:
@sanders41 I left some explanation in the description above

Reply:
This is fine with me, we may just want a note somewhere (maybe the README?) about this. I'm guessing we will get questions about it since it is different and people could get "errors" if they needed to run the build first now.

Reply:
oh you're totally right....`nox -s dev` will automatically run it, but everywhere that has `docker compose up webserver` it won't exist...

Reply:
I'm going to fix all of those new docs issues in the other docs PR #981